ci/recipe/meta.yaml

```diff
@@ -18,6 +18,7 @@ requirements:
   host:
     - python  {{ python }}
     - setuptools
+    - pytest
```

Comment:
Please add a comment explaining why this dep is necessary (just so that future readers don't remove this because they think its a mistake and remove it - I think this is the only case where we specify pytest as a direct dependency for a plugin).